msal/tests/dev apps/XForms/XForms.Android/MainActivity.cs

```diff
@@ -47,13 +47,14 @@ protected override void OnCreate(Bundle bundle)
             global::Xamarin.Forms.Forms.Init(this, bundle);
             LoadApplication(new App());
 
+            // To activate system webview, remove '//' from line 51 below, and comment out line 57 ;
             App.UIParent = new UIParent(this);
 
             #region Web browsers for MSAL.NET Android
-            // To activate embedded webview, remove '//' from line 56 below, 
-            // and comment out line 50 above -> App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
+            // To activate embedded webview, remove '//' from line 57 below, 
+            // and comment out line 51 above -> App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
```

Comment:
keeping these numbers in check... is difficult. code should be refactored to make this easier to understand and setup.... ideally this should be controlled by a setting in the test app.

Reply:
I'll resolve in a follow up PR

Reply:
ok.

---
In reply to: [224643193](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1235#discussion_r224643193) [](ancestors = 224643193)